include/swift/Parse/Parser.h

```diff
@@ -1976,11 +1978,8 @@ class Parser {
   ParserResult<Stmt> parseStmtDiscard();
   ParserResult<Stmt> parseStmtDefer();
   ParserStatus
-  parseStmtConditionElement(SmallVectorImpl<StmtConditionElement> &result,
-                            Diag<> DefaultID, StmtKind ParentKind,
-                            StringRef &BindingKindStr);
-  ParserStatus parseStmtCondition(StmtCondition &Result, Diag<> ID,
-                                  StmtKind ParentKind);
+  parseStmtConditionElement(SmallVectorImpl<StmtConditionElement> &result, Diag<> DefaultID, StmtKind ParentKind, StringRef &BindingKindStr);
```

Comment:
Small thing: Not sure if it is a strict rule, but the code here all seems to be <= 80 characters per line. Maybe keep the formatting as it was before? Would also clean up the diff a little.

Reply:
This should be all automated, see [`How do I format my changes?` section in FAQ.md](https://github.com/apple/swift/blob/main/docs/HowToGuides/FAQ.md#how-do-i-format-my-changes)

Reply:
Nice, didn't know about that, thanks!

Reply:
Thanks, I had missed the automatic format tool!